app/views/about/index.html.erb

```diff
@@ -126,7 +126,7 @@
         <li><%= link_to _('Wiki'), external_link_path(type: 'wiki'), :rel => "external" %></li>
       </ul>
       <h6><%= _("IRC") %></h6>
```

Comment:
```suggestion
      <h6><%= _("Matrix") %></h6>
```

Shouldn't the heading also be changed? 🙂

Reply:
ah, good catch, updated